scripts/s3_upload.py

```diff
@@ -157,6 +157,79 @@ def get_key(job_cnf, key):
     return job_config
 
 
+def wait_for_data_availability(
```

Comment:
I think it's a really bad idea to run these types of operations in this repository. This repo is supposed to be for just compressing and transferring the data to the cloud as quickly as possible. I was reluctant to add the command to send the rest request to initiate a code ocean capsule/pipeline. If we start add polling to check the status of a running code ocean capsule, it can potentially block other people uploading data.

Reply:
This validation is not happening in the HPC nodes but in the virtual machine if we are worried about blocking people there. In terms of the creation, it's important that we create the data asset before since we are not using the trigger capsule anymore but just manually switching the data asset in the SmartSPIM pipeline.

Reply:
Is it not possible to have a capsule at the start of the pipeline that registers the data asset? If that's not possible, after the data is successfully uploaded, we should probably send a REST request to an AWS lambda function to perform the step of registering the data asset and triggering a pipeline.

Reply:
Ohh yeah, we can totally do that in the pipeline. That's why I set up a meeting with you to figure out the credentials without the .env file a while ago. We agreed in using the .env. However, I avoided using it since we're sharing the pipeline with a lot of people (e.g., we shared the pipeline today with folks from code ocean to figure out a bug).

Maybe, when I am back from my vacations, we can figure it out. That would be super helpful! 😊

Reply:
Ohh yeah, we can totally do that in the pipeline. That's why I set up a meeting with you to figure out the credentials without the .env file a while ago. We agreed in using the .env. However, I avoided using it since we're sharing the pipeline with a lot of people (e.g., we shared the pipeline today with folks from code ocean to figure out a bug).

Maybe, when I am back from my vacations, we can figure it out. That would be super helpful! 😊

Reply:
Sounds good